formats.go

```diff
@@ -25,6 +26,7 @@ var Formats = map[string]func(interface{}) bool{
 	"time":                  isTime,
 	"hostname":              isHostname,
 	"email":                 isEmail,
+	"phone":                 isPhone,
```

Comment:
To avoid confusion what do you think about `phone-number` or `tel` (which matches the HTML5 tel input type)?

Reply:
Agree, renamed to tel